UniShed/schedule/models.py

```diff
@@ -47,35 +51,21 @@ class TimesOfStudy(models.TextChoices):
         SEVENTH_CLASS_TIME = '18:00-19:30', _('7.ПАРА 18:00-19:30')
         EIGHTH_CLASS_TIME = '19:40-21:10', _('8.ПАРА 19:40-21:10')
 
-    times_of_study = models.CharField(max_length=30, choices=TimesOfStudy.choices)
-    shift_of_times = models.ForeignKey(Shift, on_delete=models.CASCADE, related_name="Times")
-
-
-class CourseStudent(models.Model):
-    degree_name = models.ForeignKey("univer_structure.Degree", on_delete=models.CASCADE, related_name='+')
-    course_number = models.ForeignKey("univer_structure.CourseNumber", on_delete=models.CASCADE, related_name="+")
-    group_number = models.ForeignKey("univer_structure.Group", on_delete=models.CASCADE, related_name="+")
-
-
-class LectorAttachedDiscipline(models.Model):
-    lector = models.ForeignKey("univer_structure.Lector")
-    discipline = models.ForeignKey("univer_structure.Discipline")
-
+    class DayOfStudy(models.TextChoices):
+        MONDAY = 'Понедельник', _('Пн')
+        TUESDAY = 'Вторник', _('Вт')
+        WEDNESDAY = 'Среда', _('Ср')
+        THURSDAY = 'Четверг', _('Чт')
+        FRIDAY = 'Пятница', _('Пт')
+        SATURDAY = 'Суббота', _('Сб')
 
-class CourseScheduleItem(models.Model):
+    day = models.CharField(max_length=20, choices=DayOfStudy.choices)
+    times_of_study = models.CharField(max_length=30, choices=TimesOfStudy.choices)
     auditorium_of_course = models.ForeignKey("univer_structure.Auditorium", on_delete=models.CASCADE,
```

Comment:
just `auditorium` is enough here. It is already under CourseScheduleItem so the extra `of_course` is not necessary and just makes the field name too long